packages/alfa-result/src/result.ts

```diff
@@ -59,8 +59,14 @@ export interface Result<T, E = T>
   andThen<U, F>(result: Mapper<T, Result<U, F>>): Result<U, E | F>;
   or<U, F>(result: Result<U, F>): Result<T | U, F>;
   orElse<U, F>(result: Thunk<Result<U, F>>): Result<T | U, F>;
-  get(message?: string): T;
-  getErr(message?: string): E;
+  /**
+   * This may throw an exception. Use only when you can provide an external
+   * guarantee that it won't be used on Err. E.g. in tests, or when some
+   * condition exists that TypeScript cannot see (document it!)
+   *
+   * @internal
+   */
+  getUnsafe(message?: string): T;
```

Comment:
**Suggestion:** We should similarly have a `getErrUnsafe` to avoid type assertions and provide a fallback of the previous behavior.

Reply:
Make sense, added.